Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs

```diff
@@ -412,4 +487,14 @@ private void GlobalParams_Collapsed(object sender, RoutedEventArgs e)
             Row2GlobalParams.Height = new GridLength(35);
         }
     }
+    public class ParamEventArgs
+    {
+        private List<AppModelParameter> mModelParamsList;
+        public ParamEventArgs(List<AppModelParameter> modelParamsList)
+        {
+            mModelParamsList = modelParamsList;
+        }
+
+        public List<AppModelParameter> ModelParamsList { get { return mModelParamsList; } }
+    }
```

Comment:
Avoid placing Class inside other class, please save it as seperate class cs file (first check if this event args really needed- see other comments)

Reply:
Done - separated to different class
But why to avoid placing class inside other class?